packages/components/src/search/autocompleters/countries.js

```diff
@@ -23,6 +23,9 @@ export default {
 	options() {
 		return wcSettings.dataEndpoints.countries || [];
 	},
+	getSearchExpression( query ) {
+		return '^' + query;
```

Comment:
My initial impression was that we should match not only the first word but any word in a country name.
```suggestion
		return '^.*\\b' + query;
```
So searching for `Salvador` or `Korea` would return results, which will not appear if we only test the first word of the country name.

The drawback is that other queries would return irrelevant results (for example, when searching `th`):
![image](https://user-images.githubusercontent.com/3616980/51601503-d2e40580-1f04-11e9-98af-b57605650f1c.png)

So, thinking it twice, I think your solution is better.

Reply:
Thanks for the review, @Aljullu!  

>My initial impression was that we should match not only the first word but any word in a country name.

🤔 I like that solution too.  I'm a bit torn here as to which would be a better option.  I'll go ahead and get this merged for now and we can adjust later if we need to.